torchx/schedulers/aws_batch_scheduler.py

```diff
@@ -455,6 +462,12 @@ def _cancel_existing(self, app_id: str) -> None:
     def _run_opts(self) -> runopts:
         opts = runopts()
         opts.add("queue", type_=str, help="queue to schedule job in", required=True)
+        opts.add(
+            "user",
+            type_=str,
+            default=getpass.getuser(),
+            help="The username to tag the job with. `getpass.getuser()` if not specified.",
+        )
```

Comment:
I wonder if we want to add a more general concept of identity to TorchX that we could allow overriding with entrypoints etc but I think this is fine for now